### ISSUE TITLE
Accessibility issues with the embed code

### ISSUE THREAD

**Issue by mfairchild365** [unl/UNL_MediaHub#260]
It looks like there are still some accessibility issues with the embed script.  These issues will cause webaudit failures and should be fixed asap.

To identify them you can use the SiteMaster Pa11y bookmarklet.
